src/toolbelt/core.cljc

```diff
@@ -125,9 +127,9 @@
     (loop [x  (first coll)
            xs (rest coll)]
       (cond
-        (pred x)    x
+        (pred x) x
```

Comment:
Looks like your editor un-aligned the indentation on all of these.

Reply:
Unfortunately yes :(